environment.yml

```diff
@@ -0,0 +1,296 @@
+# This file may be used to create an environment using:
+# $ conda create --name <env> --file <this file>
+# platform: linux-64
+@EXPLICIT
```

Comment:
Use of a "spec file" in this way is new to me!

It's worth being explicit about intended setup - i naively ran "conda env create -n gear" which threw an error - it's probably better not to call the spec file `environment.yml`? 

@jmarshrossney has put a lot of thought into environment management with complex dependencies as a form of PhD procrastination, and may have suggestions here

 

https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#building-identical-conda-environments

Reply:
Thanks for tagging me. It is true that I have often gazed into the hellscape that is python packaging as a form of ritual self punishment for my waning interest in the research I was doing.

As far as I know `conda list --explicit` is good unless you're installed anything using `pip`, but as @metazool says should probably be in a `packages.txt` of something instead of an `environment.yml`.

Beyond that, happy to look at this together at a time convenient to both of us, in a new PR :)